test/NuGet.Clients.Tests/NuGet.CommandLine.Test/MSBuildUtilityTest.cs

```diff
@@ -333,6 +333,21 @@ public void GetMsbuildDirectoryFromPath_PATHENVWithQuotes_Succeeds()
             }
         }
 
+        [Fact]
+        public void CombinePathWithVerboseError_CombinesPaths()
+        {
+            var paths = new[] {"C:\\", "directory/", "\\folder", "file.txt"};
+            Assert.Equal(Path.Combine(paths), MsBuildUtility.CombinePathWithVerboseError(paths));
+        }
+
+        [Fact]
```

Comment:
This should make the test run on Windows only:

```suggestion
        [PlatformFact(Platform.Windows)]
```

Reply:
Thanks, @jeffkl. Looks like they passed on Linux, too, so I'll add that to the list.

Reply:
I don't think the test was running on Linux and for some reason its still running and failing on Mac.  Let's try Windows only as that still validates the `try...catch` logic and the functionality isn't OS specific so that should be fine.